build.sbt

```diff
@@ -47,7 +47,7 @@ lazy val root = project
   .enablePlugins(ScalaJSPlugin)
   .settings(skip in publish := true)
   .settings(historyPath := None)
-  .aggregate(coreJVM, coreJS, http4s, catsInteropJVM, catsInteropJS)
+  .aggregate(coreJVM, coreJS, zqueryJVM, zqueryJS, http4s, catsInteropJVM, catsInteropJS)
```

Comment:
If we leave it like that, it is going to be published under `com.github.ghostdogpr` as `zquery` with caliban's version number. If you later move it to its own repo, it might be confusing for people who might depend on the old one.

How about moving it inside caliban core for now? That way it won't be published and there won't be any ambiguity later when it becomes a library.

Reply:
I added `skip in publish := true` for `zquery`. Does that work?

Reply:
Hmm it won’t be published but it won’t be part of caliban either, so I think it’ll fail when you try to use it?

Reply:
Ah, I see. Done now.